recipes/blastlegacy/meta.yaml

```diff
@@ -0,0 +1,31 @@
+package:
+  name: blast-legacy
+  version: "2.2.22"
+
+# These are compiled binaries. More are available at 
+# ftp://ftp.ncbi.nlm.nih.gov/blast/executables/release/2.2.22/ 
+source:
+  fn: blast-2.2.22-x64-linux.tar.gz                                                                       # [linux64]
+  url: ftp://ftp.ncbi.nlm.nih.gov/blast/executables/release/2.2.22/blast-2.2.22-x64-linux.tar.gz          # [linux64]
+  fn: blast-2.2.22-universal-macosx.tar.gz                                                                # [osx]
+  url: ftp://ftp.ncbi.nlm.nih.gov/blast/executables/release/2.2.22/blast-2.2.22-universal-macosx.tar.gz   # [osx]
+  fn: blast-2.2.22-x64-win64.exe                                                                          # [win64]
+  url: ftp://ftp.ncbi.nlm.nih.gov/blast/executables/release/2.2.22/blast-2.2.22-x64-win64.exe             # [win64]
+
+build:
+  number: 0
+  script:
+    - mkdir -p "$PREFIX/bin"                                  # [osx or linux64]
+    - mv -v $SRC_DIR/bin/* $PREFIX/bin                        # [linux64]
+    - ditto --rsrc -v --arch i386 $SRC_DIR/bin/* $PREFIX/bin  # [osx]
```

Comment:
Can you please add a comment mentioning the conda issue that this works around? Thanks!